data/mods/Aftershock/effects_on_condition.json

```diff
@@ -44,7 +44,21 @@
   },
   {
     "type": "effect_on_condition",
-    "id": "afs_motion_sensor_eoc_end_ping",
-    "effect": [ { "u_add_var": "afs_motion_sensor_ping", "type": "flag", "context": "motion_sensor", "value": "0" } ]
```

Comment:
It looks like you accidentally erased this one.

Reply:
Must have been during a rebase, restored.